BinaryPartAbstractImage error: java.net.MalformedURLException: no protocol

When using Java 6 on linux I cannot load an image. It appears to work fine on windows.

```
Caused by: org.docx4j.openpackaging.exceptions.Docx4JException: Error checking image format
        at org.docx4j.openpackaging.parts.WordprocessingML.BinaryPartAbstractImage.ensureFormatIsSupported(BinaryPartAbstractImage.java:429)
        at org.docx4j.openpackaging.parts.WordprocessingML.BinaryPartAbstractImage.ensureFormatIsSupported(BinaryPartAbstractImage.java:331)
        at org.docx4j.openpackaging.parts.WordprocessingML.BinaryPartAbstractImage.createImagePart(BinaryPartAbstractImage.java:225)
        ... 
Caused by: java.net.MalformedURLException: no protocol: /tmp/img8224977656539125863.img
        at java.net.URL.<init>(URL.java:567)
        at java.net.URL.<init>(URL.java:465)
        at java.net.URL.<init>(URL.java:414)
        at org.docx4j.openpackaging.parts.WordprocessingML.BinaryPartAbstractImage.ensureFormatIsSupported(BinaryPartAbstractImage.java:421)
        ... 
```

I believe the URI should be initialised with a `file://` protocol prefix.

From `BinaryPartAbstractImage.java` line 415:

``` java
                fos.close();
                fos = null;

                // We need to refresh image info 
                imageManager.getCache().clearCache();
                info = getImageInfo(new URL(imageFile.getAbsolutePath()));

                // Debug ...
                displayImageInfo(info);
```

e.g.

``` java
                fos.close();
                fos = null;

                // We need to refresh image info 
                imageManager.getCache().clearCache();
                info = getImageInfo(new URL("file://" + imageFile.getAbsolutePath()));

                // Debug ...
                displayImageInfo(info);
```

Unfortunately I'm not near a machine where I can build and verify this fix (yet)

--- a/src/main/java/org/docx4j/openpackaging/parts/WordprocessingML/BinaryPartAbstractImage.java
+++ b/src/main/java/org/docx4j/openpackaging/parts/WordprocessingML/BinaryPartAbstractImage.java
@@ -418,7 +418,7 @@ public abstract class BinaryPartAbstractImage extends BinaryPart {
 				
 				// We need to refresh image info 
 				imageManager.getCache().clearCache();
-                info = getImageInfo(new URL(imageFile.getAbsolutePath()));
+                info = getImageInfo(new URL("file://" + imageFile.getAbsolutePath()));
 				
 				// Debug ...
 				displayImageInfo(info);

